landoapi/transplants.py

```diff
@@ -301,6 +303,47 @@ def warning_wip_commit_message(*, revision, **kwargs):
         return "This revision is marked as a WIP. Please remove `WIP:` before landing."
 
 
+@RevisionWarningCheck(8, "Repository is under a soft code freeze.", True)
+def warning_code_freeze(*, repo, **kwargs):
+    repo_fields = repo.get("fields", {})
+    short_name = repo_fields.get("shortName")
+    if short_name == "mozilla-central":
+        message = ""
+        try:
+            product_details = requests.get(
+                "https://product-details.mozilla.org/1.0/firefox_versions.json"
+            ).json()
```

Comment:
This value should probably be also stored in the `Repo` object -- in case we want to apply different product URLs to different repos/products (e.g. [comm-central] (https://product-details.mozilla.org/1.0/thunderbird.json) or different URLs in DEV/PROD/local.